README.md

```diff
@@ -27,41 +27,44 @@ gardenClusters:
 ```
 
 ## Usage
-An example `kubeconfig` for a shoot cluster could look like the following:
+An example `kubeconfig` for a shoot cluster looks like the following:
 
 ```yaml
+# supported with kubectl version v1.20.0 onwards
 apiVersion: v1
 kind: Config
 clusters:
-  - name: shoot--myproject--mycluster
-    cluster:
-      server: https://api.mycluster.myproject.example.com
-      certificate-authority-data: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCi4uLgotLS0tLUVORCBDRVJUSUZJQ0FURS0tLS0t
-      extensions:
-        - name: client.authentication.k8s.io/exec
-          extension:
-            shootRef:
-              namespace: garden-myproject
-              name: mycluster
-            gardenClusterIdentity: landscape-dev # must match with the garden cluster identity from the config
+- name: shoot--myproject--mycluster
+ cluster:
+   server: https://api.mycluster.myproject.example.com
+   certificate-authority-data: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCi4uLgotLS0tLUVORCBDRVJUSUZJQ0FURS0tLS0t
+   extensions:
+   - name: client.authentication.k8s.io/exec
+    extension:
+      shootRef:
+        namespace: garden-myproject
+        name: mycluster
+      gardenClusterIdentity: landscape-dev # must match with the garden cluster identity from the config
 contexts:
-  - name: shoot--myproject--mycluster
-    context:
-      cluster: shoot--myproject--mycluster
-      user: shoot--myproject--mycluster
+- name: shoot--myproject--mycluster
+ context:
+   cluster: shoot--myproject--mycluster
+   user: shoot--myproject--mycluster
```

Comment:
You broke the indentation of this YAML snippet somehow.

Reply:
oops, good catch